bin/utest/vm_map.c

```diff
@@ -66,3 +68,33 @@ int test_sharing_memory_child_and_grandchild(void) {
   assert(munmap(map, pgsz) == 0);
   return 0;
 }
+
+int test_anon_private_simple(void) {
+  size_t pgsz = getpagesize();
+  char *map =
+    mmap(NULL, pgsz, PROT_READ | PROT_WRITE, MAP_ANON | MAP_PRIVATE, -1, 0);
+
+  assert(map != (char *)MAP_FAILED);
+
+  strcpy(map, "Hello, World!");
+
+  pid_t pid = fork();
+
+  assert(pid >= 0);
+
+  if (pid == 0) {
+    /* child */
+    assert(strcmp(map, "Hello, World!") == 0);
+    strcpy(map, "Hello from child!");
+    assert(strcmp(map, "Hello from child!") == 0);
+    exit(0);
+  } else {
+    /* parent */
+    wait_for_child_exit(pid, 0);
+
+    assert(strcmp(map, "Hello, World!") == 0);
+  }
+
+  assert(munmap(map, pgsz) == 0);
+  return 0;
+}
```

Comment:
That test doesn't work on AArch64 - you get infinity loop  in pmap. Please fix that or explain what is a source of that problem.

Reply:
I cannot reproduce this problem. Could you provide some example of that case? Or what can I do to observe this situation?

Reply:
```
git clone https://github.com/bkjg/mimiker
cd mimiker
git checkout copy-on-write
make -j4 BOARD=rpi3
./launch --board rpi3 -d test=user_anon_private_simple
```

Reply:
That was because of the bug with permissions. It's fixed now.